dask_ml/wrappers.py

```diff
@@ -11,6 +11,7 @@
 from ._partial import fit
 from ._utils import copy_learned_attributes
 from .metrics import get_scorer, check_scoring
+from sklearn.metrics import get_scorer as sklearn_get_scorer
```

Comment:
Slight preference for just using `import sklearn.metrics` and then `sklearn.metrics.get_scorer` (we'll soon have more space to work with :)